target/linux/ath79/dts/qca9558_engenius_emr3000.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "EnGenius EMR3000";
+	compatible = "engenius,emr3000", "qca,qca9558";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "emr3000:white:power";
+			gpios = <&gpio 18 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		offline {
+			label = "emr3000:amber:offline";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+
+		rssi {
+			label = "emr3000:orange:rssi";
```

Comment:
Please either use amber or orange for both leds. amber is preferred.

Reply:
How does having only one rssi LED make sense?

Reply:
Thanks for comment. Replace it to WPS